keras/losses.py

```diff
@@ -1,19 +1,34 @@
 from __future__ import absolute_import
 import six
 from . import backend as K
-from .utils.generic_utils import deserialize_keras_object
+from .utils.generic_utils import (deserialize_keras_object,
+                                  serialize_keras_object)
 
 
 # noinspection SpellCheckingInspection
 def mean_squared_error(y_true, y_pred):
+    """Mean squared error (MSE)
+
+    A model which minimizes the mean squared error provides an estimate
```

Comment:
It's a loss function, not a model. Also, I don't think this docstring provides any added value compared to the name of the function alone.